spaces/S000016/properties/P000142.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000016
+property: P000142
+value: false
+---
+
+An infinite subset $A\subseteq X$ has {S15} as subspace. Since {S15|P3}, it follows that the compact Hausdorff subsets of $X$ are the finite subsets, which have the discrete topology. Therefore if $A\subseteq X$ then $A\cap K$ is open in $K$ for all Hausdorff compact $K$. Taking any $A$ which isn't open shows that $X$ is not a $k_3$-space.
```

Comment:
```suggestion
An infinite subset $A\subseteq X$ contains {S15} as a subspace. Since {S15|P3}, it follows that the compact Hausdorff subsets of $X$ are the finite subsets, which have the discrete topology. Therefore if $A\subseteq X$ then $A\cap K$ is open in $K$ for all Hausdorff compact $K$. Taking any $A$ which isn't open shows that $X$ is not a $k_3$-space.
```